s/sudo

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Query admin rights http://stackoverflow.com/a/21295806/1641422
+fsutil dirty query $SYSTEMDRIVE >/dev/null 2>&1
+ret=$?
+if [ $ret -eq 127 ]; then
+	# If fail try another way http://stackoverflow.com/a/28268802
+	fltmc >/dev/null 2>&1
+	ret=$?
+fi
+if [ $ret -eq 0 ]; then
+	#if elevated just run command
+	exec $@
```

Comment:
the only real change is here in using `$@` instead of the `$command_to_run` . since the variable is not needed, the whole block can move up.

Reply:
This should perhaps be spelled `exec "$@"` rather than `exec $@`, I guess?

Reply:
Yes thx, fixed now.

Confirmed: `sudo  bash -c 'echo $#' 0 1 2`  and `bash -c 'echo $#' 0 1 2` give the same result (2) now.

The difference seems to be quite subtle (a bit against my initial, wrong intuition)